internal/proto/buf.yaml

```diff
@@ -0,0 +1,14 @@
+version: v1
+name: buf.build/conduitio/conduit-processor-sdk
```

Comment:
We should upload this Buf module to the Buf schema registry, so that people can use the proto file for plugins in other languages (or we can use it to build other SDKs). That's why I'd rather use this folder just for Buf and move the Go code to a separate package (or simply attach the methods onto commands, as we now do in [conduit-commons](https://github.com/ConduitIO/conduit-commons/blob/680d445703c04b1df437008ae312e7b9bf838526/opencdc/proto.go#L38)).